cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -75,8 +75,9 @@ if node['cdo-apps']['daemon']
   cronjob at:'55 8 * * *', do:deploy_dir('bin', 'cron', 'ops_data_pull')
   cronjob at:'45 5 * * *', do:deploy_dir('bin', 'cron', 'admin_stats')
   cronjob at:'40 4 * * *', do:deploy_dir('bin', 'cron', 'funometer')
-  cronjob at: '30 6 * * *', do: deploy_dir('bin', 'cron', 'admin_progress')
-  cronjob at: '10 7 * * *', do: deploy_dir('bin', 'cron', 'retention_stats')
+  cronjob at: '30 6 * * *', do:deploy_dir('bin', 'cron', 'admin_progress')
+  cronjob at: '10 7 * * *', do:deploy_dir('bin', 'cron', 'retention_stats')
+  cronjob at: '30 9 * * *', do:deploy_dir('bin', 'cron', 'update_dotd') 
```

Comment:
maybe move this up to the staging section since we don't need it to run everywhere -- also check if staging's clock is UTC